IronGram/build/resources/main/static/index.html

```diff
@@ -18,6 +18,8 @@
 <form action="/upload" method="post" enctype="multipart/form-data" id="upload" hidden>
     <input type="text" placeholder="Receiver Name" name="receiverName"/>
     <input type="file" name="photo">
+    <input type="number" name="sleepTime" placeholder="Enter viewable time (seconds)">
```

Comment:
While the user can't really see this sleepTime isn't a great name for the variable.  It gives clues as to how you delete and isn't super descriptive for the form itself.